Apps/phcuckoo/cuckoo_connector.py

```diff
@@ -403,6 +439,28 @@ def _handle_detonate_url(self, param):
 
         return self._poll_for_task(action_result, task_id, key=url)
 
+    def _handle_submit_strings(self, param):
+        action_result = self.add_action_result(ActionResult(dict(param)))
+        ret_val = self._check_version(action_result)
+        if phantom.is_fail(ret_val):
+            return ret_val
+
+        strings = "\n".join(map(lambda x: x.strip(), param['url'].split()))
+        strings = "\n".join(map(lambda x: x.strip(), strings.split(",")))
+        strings = list(map(lambda x: x.strip(), strings.split("\n")))
+
+        if len(strings) == 0:
```

Comment:
Can you make this line and the if on line 455 into one if statement?

Reply:
Fixed